content/news/2002-09-11-the-legalities-of-open-source-and-linux.md

```diff
@@ -0,0 +1,16 @@
+Title: The Legalities of Open Source and Linux
+Tags: general meeting
+Event: 2002-09-11 7:30 pm to 9:00 pm
+Speaker: Larry Rosen
+Location: O'Reilly and Associates
+Author: Eric Eisenhart
+
+Our speaker this month is an expert on legal issues relating to open source software and intellectual property; instead of a specific pre-planned formal presentation, we've decided to do a Q&A session, so read up, think of some questions (interesting, hard, you're just curious, whatever...) and bring them to the meeting with you. Or you can email [legal-questions@nblug.org](mailto:legal-questions@nblug.org) and we can ask your questions for you.
+
+Larry Rosen is the founding partner of [ROSENLAW.COM LLP](http://www.rosenlaw.com/), a law firm that specializes in intellectual property matters for large and small companies and individuals. He is also the general counsel and executive director of [Open Source Initiative](http://www.opensource.org/), an organization that reviews and approves licenses for open source software. His articles on legal issues relating to open source appear monthly in [Linux Journal](http://www.linuxjournal.com/search.php?author=LawrenceRosen). Before he became an attorney, Larry taught computer programming and managed several computer departments at [Stanford University](http://www.stanford.edu/). He then worked in private industry, managing computer and applications development. At one large firm he guided the design, implementation, manufacturing and marketing of successful products for data and voice communications.
+
+[http://www.opensource.org/](http://www.opensource.org/)
+
+[http://www.rosenlaw.com/html/lawrenceerosen.html](http://www.rosenlaw.com/html/lawrencerosen.html)
+
+[http://www.linuxjournal.com/search.php?author=LawrenceRosen](http://www.linuxjournal.com/search.php?author=LawrenceRosen)
```

Comment:
Sure thing. I simply copied the links, I didn't know if we should update them or not.